go-controller/pkg/ovn/ovn.go

```diff
@@ -104,7 +119,13 @@ func (oc *Controller) WatchEndpoints() {
 				logrus.Errorf("Error in adding load balancer: %v", err)
 			}
 		},
-		UpdateFunc: func(old, new interface{}) { return },
+		UpdateFunc: func(old, new interface{}) {
+			ep := new.(*kapi.Endpoints)
+			err := oc.addEndpoints(ep)
```

Comment:
For the time being, lets go with your original idea of doing this at the top. Lets re-introduce the cache once we get the "sync" feature added.